definition.tex

```diff
@@ -53,6 +53,8 @@ \section{Defining Proof-of-Burn}
 
 \import{./}{algorithms/alg.bind-game.tex}
 
+It is desired for the protocol that a burn address generated encodes one and only one tag, so that an adversary cannot create an address that commits to multiple things. Specifically, if we fix a burn address $\burnAddr$, $\BurnVerify(1^\kappa, t, \burnAddr)$ should only evaluate to $\true$ for a single tag $t$. To state this property we introduce another game-based security definition. The binding game $\bindattack$ is illustrated in Algorithm~\ref{alg.bind-game}.
```

Comment:
Avoid separating multiple math expressions by comma:

if we fix a burn address $\burnAddr$, _then_ $\BurnVerify(1^\kappa, t, \burnAddr)$ should...